source/guide_mailman.rst

```diff
@@ -0,0 +1,252 @@
+
+::
+
+ [isabell@stardust ~]$ chmod -R 0755 /var/www/virtual/isabell/mailman/cgi-bin
+
+Step 5
+------
+
+Because Mailman_ doesn't handle our .qmail-configuration automatically, we need to help it create the necessary aliases. The following script is based on the script provided in the official installation instructions and may be used by placing it in your ``~/bin`` folder (for example as ``addlist.sh``):
+
+.. code :: bash
+
+ #!/bin/sh
+ if [ $# = 1 ]; then
+ i=$1
+ echo Making links to $i in the current directory...
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman post $i" > .qmail-$i
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman admin $i" > .qmail-$i-admin
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman bounces $i" > .qmail-$i-bounces
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman confirm $i" > .qmail-$i-confirm
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman join $i" > .qmail-$i-join
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman leave $i" > .qmail-$i-leave
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman owner $i" > .qmail-$i-owner
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman request $i" > .qmail-$i-request
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman subscribe $i" > .qmail-$i-subscribe
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman unsubscribe $i" > .qmail-$i-unsubscribe
+ fi
+
+Don't forget to make the script executable:
+
+::
+
+ [isabell@stardust ~]$ chmod +x ~/bin/addlist.sh
+ [isabell@stardust ~]$
+
+After creating a list via the webinterface, you can then run this script to create the required .qmail-files (like ``addlist.sh listname`` if you stored it as ``~/bin/addlist.sh`` and want to create aliases for a list ``listname``).
+
+Configuration
+=============
+
+By now we have installed all the necessary files - let's tell them what they should actually do.
+
+Step 1
+------
+
+Create a mailbox_ for Mailman to use to send e-mails. In this example, we are going to use ``mailmanbox@isabell.uber.space``.
+
+Step 2
+------
+
+Set the following options in ``/var/www/virtual/isabell/mailman/Mailman/mm_cfg.py`` (change values accordingly!):
+
+.. code:: python
+
+ # configure default domains to use for the webinterface and e-mail addresses
+ DEFAULT_URL_HOST = 'isabell.uber.space'
+ DEFAULT_EMAIL_HOST = 'isabell.uber.space'
+
+ # configure mailmans mailbox
+ SMTP_AUTH = True
+ SMTP_USE_TLS = True
+ SMTPHOST = 'stardust.uberspace.de'
+ SMTPPORT = '587'
+
+ SMTP_USER = 'mailmanbox@isabell.uber.space'
+ SMTP_PASSWD = 'betterPWthanThis'
+
+ # tell mailman to use HTTPS
+ DEFAULT_URL_PATTERN = 'https://%s/mailman'
+
+You can look up the meaning  and default value of each variable in the file ``Defaults.py`` in the same folder.
+
+.. warning:: Do not modify the ``Defaults.py`` as it may be overwritten on updates!
+
+
+Finishing Installation
```

Comment:
Maybe include an example .htaccess to rewrite all HTTP requests to /mailman/listinfo or similar?

Reply:
Sure, why not. It's now included as "Redirecting HTTP-requests" in the finishing steps.